### ISSUE TITLE
Rate::getAbsoluteValue() returns a string

### ISSUE THREAD

**Issue by greg0ire** [mhor/php-mediainfo#65]
Sample code
```php
var_dump(current($info->getVideos())->get('height')->getAbsoluteValue());die();
```

Output:

> string(4) "1080"

I would expect an integer, as documented.

**Comment by mhor** [comment]
@greg0ire thanks for your feedback I will try to solve this soon as possible.
Cast ```$absoluteValue``` to int [here](https://github.com/mhor/php-mediainfo/blob/master/src/Attribute/Rate.php#L27) should solve the problem.

**Comment by greg0ire** [comment]
Hey there's no rush, don't worry, I just cast the return value to int ;) . Maybe find why it is a string in the first place?

**Comment by mhor** [comment]
@greg0ire I open a pull request (#65) to solve your issue.
This bug is due to mediainfo output parsing, unfortunately when xml output is transformed into array ([there](https://github.com/mhor/php-mediainfo/blob/master/src/Parser/AbstractXmlOutputParser.php#L12)) all attributes values are transformed to string.

**Comment by greg0ire** [comment]
You mean #66 ;)

**Comment by mhor** [comment]
Yes sorry :+1:

**Comment by mhor** [comment]
Thank you @greg0ire , #66 is now merged and realeased as [2.3.1](https://github.com/mhor/php-mediainfo/releases/tag/2.3.1)
